src/inmanta/server/services/orchestrationservice.py

```diff
@@ -1305,6 +1306,20 @@ async def get_diff_of_versions(
 
         return version_diff
 
+    @handle(methods_v2.get_scheduler_status, env="tid")
+    async def get_scheduler_status(
+        self,
+        env: data.Environment,
+    ) -> SchedulerStatusReport:
+        await self.autostarted_agent_manager._ensure_scheduler(env.id)
+        client = self.agentmanager_service.get_agent_client(env.id, const.AGENT_SCHEDULER_ID)
+        assert client is not None
+        status = await client.trigger_get_status(env.id)
+        assert status.code == 200
+        assert status.result is not None
+        resp = SchedulerStatusReport.model_validate(status.result["data"])
+        return resp
```

Comment:
There assert should not be asserts, but raise proper errors with a proper http code